### ISSUE TITLE
ask

### ISSUE THREAD

**Issue by anwfeb** [wagiminator/ATtiny85-TinyLoad#1]
Hi,
I have tried to make the ATtiny85-TinyLoad project but i see there is a difference, if i turn the potentiometer to the left (closed) 0% then the OLED display is 0.62A. i am using R_Shunt 0.01ohm with value ICONV 7201920 and ILCAL 0.72, when i rotate potentiometer for 1.00A or 1.50A or 2.00A i get appropriate load value (with Digital Clamp Meter) , only if I adjust left 0% potentiometer i can't get 0.00A on OLED screen and get display screen 0.62A, can you help me ?

Thank you.

Edited:
if I use R_Shunt 0.1 ohm, then the current value when the potentiometer is 0% (closed) is 0.06A with the ICONV configuration 72019200

**Comment by wagiminator** [comment]
Hi, you will never reach 0V. This is due to the limitations of the OpAmp (input offset voltage and minimum output voltage). You can get closer to 0V by either choosing a larger shunt resistor or a better OpAmp.

**Comment by anwfeb** [comment]
Yes you are right, I have replaced the opamp with a generic lmv358 and I am getting 0.00A, strangely how come I can't get 0.00A with the Original TI opAmp lmv358? do you have another way to put the potentiometer at 0% position with the opamp within 0.00A?

additional information: R_Shunt 0.01 ohm no problem, it works fine, i have tested it. only i have opAmp problem, i believe i have original products. but if you have other opamp model number better i will give it a try.

**Comment by wagiminator** [comment]
With the cheap OpAmps there is a greater scatter in terms of their parameters. The pin-compatible OPA2333 is a good choice, but you seem to have had success with the generic LMV358, so keep it!

**Comment by anwfeb** [comment]
yes after i tested 10 lmv358 generic now i have 3 chips working fine. and I also added an LC Filter before the input voltage, because my power supply is not very good it makes the resistance on the display screen always change.

I'm still hoping for some sort of calibration to set the potentiometer at 0% by getting a 0.00A screen display using my original old chip, if possible.

**Comment by anwfeb** [comment]
Hi, thank you.
now i will say all the precision when i replace with OPA2333A original parts, i have bought and tried, from 10mA to 5 Ampere i tested and the result is very precision, thank you good project and very useful.
